### PR TITLE
Adding K8s version for import clusters

### PR DESCRIPTION
**Pull request by sowmyav27**
Adding k8s version as a parameter to import cluster.

### CODE REVIEW

File: tests/validation/tests/v3_api/test_import_cluster.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
 AWS_SSH_KEY_NAME = os.environ.get("AWS_SSH_KEY_NAME")
 AWS_NODE_COUNT = int(os.environ.get("AWS_NODE_COUNT", 3))
 HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "testcustom")
+RKE_K8S_VERSION = os.environ.get("RANCHER_RKECLUSTER_K8S_VERSION",
```
Comment (bmdepesa) [inline tests/validation/tests/v3_api/test_import_cluster.py:20]: Can we rename the env var to `os.environ.get("RANCHER_RKE_K8S_VERSION"`

Comment (sowmyav27) [reply tests/validation/tests/v3_api/test_import_cluster.py]: done

**Review by bmdepesa — APPROVED** [review]
LGTM

File: tests/validation/tests/v3_api/resource/cluster.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ssh_key_path: .ssh/$AWS_SSH_KEY_NAME
+kubernetes_version: $KUBERNETES_VERSION
```
Comment (jiaqiluo) [inline tests/validation/tests/v3_api/resource/cluster.yml:2]: in the case no value is passed in, is it ok to leave the line with no value in the cluster.yml file for RKE? 
Can you try this case?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: tests/validation/tests/v3_api/resource/cluster.yml (hunk 2 above)
Comment (sowmyav27) [reply tests/validation/tests/v3_api/resource/cluster.yml:2]: @jiaqiluo If no value is passed in, default value for the RKE cli will be chosen

**Review by jiaqiluo — APPROVED** [review]
LGTM

Please add the new parameter in the Jenkins as well.

**Pull request merged**